_data/tools.yml

```
@@ -14,8 +67,201 @@
+        41
+      url: https://trino.io/episodes/41.html
+- name: Apache Iceberg
+  anchor: apache-iceberg
+  category: data-source
+  logo: /assets/images/logos/apache-iceberg.png
+  logosmall: /assets/images/logos/apache-iceberg-small.png
+  description: |
+    Apache Iceberg is a high-performance format for huge analytic tables.
+    Iceberg brings the reliability and simplicity of SQL tables to big data,
+    while making it possible for engines like Spark, Trino, Flink, Presto, Hive
+    and Impala to safely work with the same tables, at the same time.
+
+    Use an Apache Iceberg data lakehous as a data source in Trino by configuring
+    a catalog with the Apache Iceberg connector.
+  links:
+    - urltext: Apache Iceberg
+      url: https://iceberg.apache.org/
+    - urltext: Apache Iceberg connector documentation
+      url: https://trino.io/docs/current/connector/iceberg.html
+    - urltext: Interview with Iceberg creator from Trino Community Broadcast 40
+      url: https://trino.io/episodes/40.html
+# there are lots more from Trino Fest, Trino Summot .. need to figure out how many we are happy to have.. 
+- name: Apache Ignite
+  anchor: apache-ignite
+  category: data-source
+  logo: /assets/images/logos/apache-ignite.png
+  logosmall: /assets/images/logos/apache-ignite-small.png
+  description: |
+    Apache Ignite is a distributed database for high‑performance applications
+    with in‑memory speed. It scales across memory and disk across multiople
+    machines without compromise.
+
+    Use an Apache Ignite database as a data source in Trino by configuring a
+    catalog with the Apache Ignite connector.
+  links:
+    - urltext: Apache Ignite
+      url: https://ignite.apache.org/
+    - urltext: Apache Ignite connector documentation
+      url: https://trino.io/docs/current/connector/ignite.html
+    - urltext: Interview about the Ignite connector from Trino Community
+        Broadcast 46
+      url: https://trino.io/episodes/46.html
+- name: Apache Kafka
+  anchor: apache-kafka
+  category: data-source
+  logo: /assets/images/logos/apache-kafka.png
+  logosmall: /assets/images/logos/apache-kafka-small.png
+  description: |
+    Apache Kafka is an open-source distributed event streaming platform used by
+    thousands of companies for high-performance data pipelines, streaming
+    analytics, data integration, and mission-critical applications.
+
+    Use an Apache Kafka topic stream as a data source in Trino by configuring a
+    catalog with the Apache Kafka connector.
+  links:
+    - urltext: Apache Kafka
+      url: https://kafka.apache.org/
+    - urltext: Apache Kafka connector documentation
+      url: https://trino.io/docs/current/connector/kafka.html
+- name: Apache Kudu
+  anchor: apache-kudu
+  category: data-source
+  logo: /assets/images/logos/apache-kudu.png
+  logosmall: /assets/images/logos/apache-kudu-small.png
+  description: |
+    Apache Kudu is an open source distributed data storage engine that makes
```

```suggestion
    Apache Kudu is an open-source distributed data storage engine that makes
```

Not sure if this needs to be addressed in this PR but I think we should do a mass search for commonly hyphenated words and pick a side. 😄